test/groovy/org/ods/usecase/JiraUseCaseSpec.groovy

```diff
@@ -197,18 +212,18 @@ class JiraUseCaseSpec extends SpecHelper {
 
         when:
         def jiraResult = [
-            "total": 0,
-            "maxResults":1000,
-            "issues":[]
+            "total"     : 0,
+            "maxResults": 1000,
+            "issues"    : []
         ]
 
-        then: 
+        then:
         // Expect an error
         def msg = shouldFail IllegalStateException, {
             usecase.getDocumentChapterData(documentType)
         }
         assert msg.contains('No documents found') && msg.contains("JIRA")
-        
+
```

Comment:
@georgfedermann what about using the following way (we already do this elsewhere)?

```
then:
e = thrown(IllegalArgumentException)
e.message == "Error: unable to assert applicability of document type '${documentType}' for project '${project.key}' and repo '${repo.id}' in phase '${phase}'. The GAMP category '0' is not supported."
```

Reply:
this is not part of the PR. This code was here before, only the IDE did some reformatting here.
But I'll can still do a rewrite.

Reply:
@georgfedermann strange, I do not see this in my current base branch, but you're correct: this passage doesn't show green in the change set. Thanks for changing!

Reply:
the test here appears to be unfinished.
It's annotated with @Ignore and calls no method under test. This must have sneaked through some earlier PR review ...